src/coreclr/nativeaot/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -126,10 +126,6 @@ static const int tccMilliSecondsToMicroSeconds = 1000;
 static const int tccMilliSecondsToNanoSeconds = 1000000;
 static const int tccMicroSecondsToNanoSeconds = 1000;
 
-static uint32_t g_cNumProcs = 0;
```

Comment:
I think we can also delete a couple SYSCONF/SYSCTL related defines and ifdefs at the beginning of this file.

Maybe the can also be deleted from config.h.in and configure.cmake?